pkg/morph/client/constructor.go

```diff
@@ -160,8 +160,14 @@ func New(key *keys.PrivateKey, opts ...Option) (*Client, error) {
 		}
 
 		cli.endpoints = cfg.endpoints
+		for _, e := range cli.endpoints {
+			cli.client, act, err = cli.newCli(e)
+			if err != nil {
+				cli.logger.Warn("morph client connection", zap.String("endpoint", e), zap.Error(err))
+				continue
+			}
+		}
 
-		cli.client, act, err = cli.newCli(cli.endpoints[0])
```

Comment:
previously we returned `cli.endpoints[0]` connection error, now - the last one. Not sure this is an expected behavior change

Reply:
can it be important?

Reply:
Can, but not likely.

Reply:
lets leave until real problems then